doc-content/enterprise-only/project-data/migrating-legacy-designer-proc.adoc

```diff
@@ -0,0 +1,41 @@
+[id='migrating-from-legacy-designer-proc']
+
+= Migrating business processes to the new designer
+
+The legacy process designer in {CENTRAL} is deprecated in {PRODUCT} {PRODUCT_VERSION_LONG}. It will be removed in a future {PRODUCT} release. The legacy process designer will not receive any new enhancements or features. Existing users that intend to utilize the functionality in the new designer should start migrating their processes to the new designer. All new processes should be created using the new process designer.
+
+[NOTE]
+====
+The {PROCESS_ENGINE} will continue to support the execution and deployment of business processes generated with the legacy designer in to the {KIE_SERVER}. If you have a legacy business process that is functioning and that you do not intend to change, it is not mandatory to migrate to the new designer at this time.
+====
+
+Only business processes that contain nodes types that are supported in the new designer can be migrated. More node types will be supported in future versions of {PRODUCT}.
+
+[NOTE]
+====
+To enable the legacy process designer in {CENTRAL}, in the upper-right corner of the window click *Settings* -> *Roles*, select a role from the left panel, click *Editors -> Select Editor -> (Legacy) Process Designer -> Read*, and then click *Save* to save the changes.
+====
+
+.Prerequisites
+* You have an existing project that contains a business process asset that was created in the legacy process designer.
+
+.Procedure
+. In {CENTRAL}, click *Menu* -> *Design* -> *Projects*.
+. Click on the project you want to migrate, for example *{PROJECT}*.
+. Click *Ok* to open the project's asset list.
+. Click on the project's *Business Process* asset to open it in the legacy process designer.
```

Comment:
Click the.. I don't think *Business Process* should be capped and bolded - you aren't referring to the UI element specifically are you? If you are, ignore.

Reply:
Yes, I am referring to the asset column name specifically. Stet.